operations/commit_queue.go

```diff
@@ -385,6 +399,7 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 	}
 	grip.Infof("Project: %s\n", projectID)
 	grip.Infof("Type of queue: %s\n", projectRef.CommitQueue.PatchType)
+	grip.Infof("Message: %s\n", projectRef.CommitQueue.Message)
```

Comment:
won't this print the message a second time? (given it's already printed [here](https://github.com/evergreen-ci/evergreen/pull/4048/files?diff=split&w=1#diff-7f9dabe9369339d415247be60a5d1b45R77))

Reply:
ah yes, forgot about that. Removed it in the other place